keyboards/kopibeng/mnk88/info.json

```diff
@@ -0,0 +1,120 @@
+{
+    "keyboard_name": "MNK88",
+    "maintainer": "kopibeng",
+    "url": "",
+    "layouts": {
+        "LAYOUT": {
```

Comment:
```suggestion
        "LAYOUT_all": {
```

Reply:
Seems this change is breaking lint check, should it be reverted?

Reply:
No, you need to fix the lint error.

Reply:
Fixed, thanks :)